clash-prelude/src/Clash/Signal/Time.hs

```diff
@@ -0,0 +1,40 @@
+{-|
+Copyright  :  (C) 2024 QBayLogic B.V.
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
+
+User friendly utilities to express time in your digital design.
+
+-}
+module Clash.Signal.Time
+ ( DomainFrequency
+ , Microseconds
+ , Milliseconds
+ , Nanoseconds
+ , PeriodToCycles
+ , Picoseconds
+ , Seconds
+ ) where
```

Comment:
Same thing really, this order doesn't really make sense for the readers of our Haddock.